plugins/available/jump.plugin.bash

```diff
@@ -0,0 +1,14 @@
+cite about-plugin
+about-plugin 'initialize jump (see https://github.com/gsamokovarov/jump)'
+
+__init_jump() {
+  command -v jump &> /dev/null
+  if [ $? -eq 1 ]; then
+    echo -e "You must install jump before you can use this plugin"
```

Comment:
I'm not a huge fan of providing output in the case of an initialization error for missing dependencies. We've had people reporting issues with things like `ssh` or `scp` failing due to output like this.

It would be better to fail silently here.